src/modules/ROOT/pages/changelog/202303/03.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta": {
+    "releasedOn": "2023.05.30"
+  },
+  "added": [
+    {
+      "what": [
+        "a stale data error in case a workflow was updated whilst a different user is editing the old version of the same workflow."
+      ],
+      "why": [
+        "to help reduce configuration errors due to multiple people editing the same resource at the same time."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/2529cf337c991cc13a2a2e15f2e84b5a8abab4a0",
+        "https://github.com/cortezaproject/corteza/commit/03806a20ab5eaf94dc9a2a8949d707a28f5c3127"
```

Comment:
This is for a different thing; this fixed an issue with action log entries not correctly recording the created workflow/trigger. This is then a fix, not an add